app/models/maintenance_stats/issue_stats.rb

```diff
@@ -0,0 +1,29 @@
+module MaintenanceStats
+    class IssueRates < BaseStat
+        def get_stats
+            {
+                "issue_closure_rate": issue_closure_rate,
+                "closed_issue_count": closed_issues_count,
+                "open_issue_count": open_issues_count,
+            }
+        end
+
+        def total_issues_count
+            return 0 if closed_issues_count.nil? && open_issues_count.nil?
+            closed_issues_count + open_issues_count
+        end
+
+        def issue_closure_rate
+            return 100.0 if total_issues_count == 0
```

Comment:
Would it make more sense to store these as `[0, 1]` instead of `[0, 100]` since we're going with fractional values anyway?

Reply:
That's fine with me if we want to go with that scheme.